src/components/Sum.jsx

```diff
@@ -1 +1,28 @@
-// Don't forget to add default values an check types of props
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
```suggestion
const Sum = ({ a, b }) => (
```
No need to write it in multiple lines here